src/containers/Trigger/Trigger.module.css

```diff
@@ -22,3 +22,5 @@
   width: 29px;
   height: 29px;
 }
+
```

Comment:
Usually, VSCode automatically removes such extra spaces when you save the file if the format on saving is enabled.  Can you please check your editor settings?